src/inmanta/deploy/persistence.py

```diff
@@ -278,14 +271,15 @@ def error_and_log(message: str, **context: Any) -> None:
                     status=status,
                     connection=connection,
                 )
+                # TODO: do we need to set blocked, is_undefined, is_orphan here?
                 await resource.update_persistent_state(
                     is_deploying=False,
                     last_deploy=finished,
                     last_deployed_version=resource_id_parsed.version,
                     last_deployed_attribute_hash=resource.attribute_hash,
                     last_non_deploying_status=const.NonDeployingResourceState(status),
                     last_deploy_result=state.last_deploy_result if state is not None else None,
-                    **extra_datetime_fields,
+                    **extra_datetime_fields,  # type: ignore[arg-type]
```

Comment:
mypy was complaining because I added the `blocked`, `is_undefined`, `is_orphan` arguments to `update_persistent_state`

Reply:
Let's not add the ignore. We need to sync the type violation in to the baseline file so that we can keep track of it.